ramls/requests-reports.raml

```diff
@@ -0,0 +1,28 @@
+#%RAML 1.0
+title: Circulation
+version: v0.1
+protocols: [ HTTP, HTTPS ]
+baseUri: http://localhost:9130
+
+documentation:
+  - title: Circulation Business Logic API
+    content: <b>API for report generation</b>
+
+types:
+  requests: !include requests.json
+
+traits:
+  language: !include raml-util/traits/language.raml
+
+resourceTypes:
+  collection-get: !include raml-util/rtypes/collection-get.raml
+
+/circulation:
+  /requests-reports:
+    displayName: Hold shelf clearance report CSV
```

Comment:
Please remove the reference to CSV from here

Reply:
Fixed